js/src/tab.js

```diff
@@ -171,54 +164,49 @@ const Tab = (($) => {
       )
 
       if (active && isTransitioning) {
-        $(active)
-          .one(Util.TRANSITION_END, complete)
-
+        EventHandler.one(active, Util.TRANSITION_END, complete)
         Util.emulateTransitionEnd(active, TRANSITION_DURATION)
       } else {
         complete()
       }
 
       if (active) {
-        $(active).removeClass(ClassName.SHOW)
+        active.classList.remove(ClassName.SHOW)
       }
     }
 
     _transitionComplete(element, active, isTransitioning, callback) {
       if (active) {
-        $(active).removeClass(ClassName.ACTIVE)
-
-        const dropdownChild = $(active.parentNode).find(
-          Selector.DROPDOWN_ACTIVE_CHILD
-        )[0]
+        active.classList.remove(ClassName.ACTIVE)
 
+        const dropdownChild = SelectorEngine.findOne(Selector.DROPDOWN_ACTIVE_CHILD, active.parentNode)
         if (dropdownChild) {
-          $(dropdownChild).removeClass(ClassName.ACTIVE)
+          dropdownChild.classList.remove(ClassName.ACTIVE)
         }
 
         if (active.getAttribute('role') === 'tab') {
           active.setAttribute('aria-selected', false)
         }
       }
 
-      $(element).addClass(ClassName.ACTIVE)
+      element.classList.add(ClassName.ACTIVE)
       if (element.getAttribute('role') === 'tab') {
         element.setAttribute('aria-selected', true)
       }
 
-      if (isTransitioning) {
+      if (isTransitioning || !active) {
```

Comment:
It seems an unrelated change here 🤔 Maybe can you explain it ?

Reply:
This check was introduced to fix fading when no initial panel was active, but i've probably missed a test. I'll push it asap, but if you want to keep things separated, i can revert this check and submit a pr just for this case.

Reply:
Is it an issue in our current `v4-dev` branch ?

Reply:
I’ve just tested it on the v4-dev, the issue is present in the visual test page

Reply:
So I prefer if you can create a PR (which target `v4-dev`) to resolve that or if you can create an issue which describe that

Reply:
Correct. I've pushed a PR to explain and resolve the issue on `v4-dev` branch

Reply:
I've reverted this change for this PR, as it is not related